homeassistant/components/mazda/config_flow.py

```diff
@@ -0,0 +1,123 @@
+"""Config flow for Mazda Connected Services integration."""
+import logging
+
+import aiohttp
+from pymazda import (
+    Client as MazdaAPI,
+    MazdaAccountLockedException,
+    MazdaAuthenticationException,
+)
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_EMAIL, CONF_PASSWORD, CONF_REGION
+from homeassistant.helpers import aiohttp_client
+
+# https://github.com/PyCQA/pylint/issues/3202
+from .const import DOMAIN  # pylint: disable=unused-import
+from .const import MAZDA_REGIONS
+
+_LOGGER = logging.getLogger(__name__)
+
+DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_EMAIL): str,
+        vol.Required(CONF_PASSWORD): str,
+        vol.Required(CONF_REGION): vol.In(MAZDA_REGIONS),
+    }
+)
+
+
+class MazdaConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for Mazda Connected Services."""
+
+    VERSION = 1
+    CONNECTION_CLASS = config_entries.CONN_CLASS_CLOUD_POLL
+
+    async def async_step_user(self, user_input=None):
+        """Handle the initial step."""
+        errors = {}
+
+        if user_input is not None:
+            await self.async_set_unique_id(user_input[CONF_EMAIL].lower())
+            self._abort_if_unique_id_configured()
```

Comment:
If the user initiates a config flow, we should not abort if a unique ID is configured. We do this so we can also override ignored entries.

Home Assistant will automatically remove the old one and have the new one replace it.